app/views/data_exports/_table.html.erb

```diff
@@ -0,0 +1,121 @@
+<% if @data_exports.count.positive? %>
+  <table class="min-w-full table-fixed dark:divide-slate-600">
+    <thead class="text-slate-700 bg-slate-50 dark:bg-slate-700 dark:text-slate-400">
+      <tr>
+        <th scope="col" class="px-6 py-3 text-left"><%= t(:"data_exports.index.table_header.id") %></th>
+        <th scope="col" class="px-6 py-3 text-left"><%= t(:"data_exports.index.table_header.name") %></th>
+        <th scope="col" class="px-6 py-3 text-left"><%= t(:"data_exports.index.table_header.type") %></th>
+        <th scope="col" class="px-6 py-3 text-left"><%= t(:"data_exports.index.table_header.status") %></th>
+        <th scope="col" class="px-6 py-3 text-left"><%= t(:"data_exports.index.table_header.created_at") %></th>
+        <th scope="col" class="px-6 py-3 text-left"><%= t(:"data_exports.index.table_header.expires_at") %></th>
+        <th scope="col" class="px-6 py-3 text-right"><%= t(:"data_exports.index.table_header.action") %></th>
+      </tr>
+    </thead>
+    <tbody
+    id="data-exports-table-body"
+    class="
+      bg-white
+      divide-y
+      divide-slate-200
+      dark:bg-slate-800
+      dark:divide-slate-700
+    ">
+      <% @data_exports.each do |data_export| %>
+        <tr
+          id="<%= dom_id(data_export) %>"
+          class="text-sm font-normal text-slate-500 dark:text-slate-400"
+        >
+          <td class="px-6 py-4">
+            <div class="text-slate-900 dark:text-slate-100 font-semibold hover:underline">
```

Comment:
**question:** Why do you have `hover:underline` on a div?

Reply:
Because I got ahead of myself :D! I've removed underline and they will be links in the next PR. 9029f204cc69a4f4266723ff8dd560e57c2368c5

Reply:
Thanks ;)